update frontend pekerja

taufikh01 opened the pull request.

taufikh0112 committed:
update frontend pekerja

taufikh0112 committed:
update list pekerjaan

Suyatna commented:
Coy, sebelum panggil
 `@FormUrlEncoded
    @POST("pencari_kerja/login")
    Call<PencariKerja> loginRequest(@Field("email") String email, @Field("password") String password);`

buat dlu model nya

zuyatna committed:
can open job detail

taufikh0112 committed:
update

tambah package "fragment"di package pekerja

taufikh0112 committed:
edit layout register pekerja

taufikh0112 committed:
update layout

taufikh0112 committed:
Update RegisterPemberiKerjaActivity.java

taufikh0112 committed:
Update activity_apply_pekerjaan.xml

taufikh0112 committed:
Update activity_post_pekerjaan.xml

taufikh0112 committed:
Update rectangle_2.xml

zuyatna committed:
fixing gradle and XLM error

taufikh01 commented:
wkwk oke siap suyy

Pada tanggal Kam, 11 Jul 2019 pukul 18.39 Suyatna <notifications@github.com>
menulis:

> Coy, sebelum panggil
> @FormUrlEncoded @POST("pencari_kerja/login") Call<PencariKerja>
> loginRequest(@Field("email") String email, @Field("password") String
> password);
>
> buat dlu model nya
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Suyatna/Mppl-JobHeist/pull/1?email_source=notifications&email_token=ALGBZ57LM5ASDHON5545TK3P64LXXA5CNFSM4IA22FH2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODZWNVEI#issuecomment-510450321>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALGBZ5ZD6QHLTUOL6DLNGYLP64LXXANCNFSM4IA22FHQ>
> .
>

taufikh0112 committed:
Merge branch 'master-2' of https://github.com/Suyatna/Mppl-JobHeist into master-2

taufikh0112 committed:
Revert "Merge branch 'master-2' of https://github.com/Suyatna/Mppl-JobHeist into master-2"

This reverts commit 4ae788995100e2f67b7d8179e2197a0213940e87, reversing
changes made to 3d9439a6375b083f77a42c6cb207777604690d8b.

taufikh0112 committed:
add all layout admin

memperbaiki layout yg kurang sesuai dan menambahkan semua layout admin sesuai dengan desain yg sudah dibuat di figma

taufikh0112 committed:
can register akun pencari kerja

zuyatna committed:
Merge remote-tracking branch 'origin/master-taufik' into master-2

zuyatna committed:
adding profiling

zuyatna committed:
final, can change account
